sparkmagic/sparkmagic/serverextension/handlers.py

```diff
@@ -38,6 +39,12 @@ def post(self):
             username = self._get_argument_or_raise(data, 'username')
             password = self._get_argument_or_raise(data, 'password')
             endpoint = self._get_argument_or_raise(data, 'endpoint')
+            auth_type = self._get_argument_if_exists(data, 'auth_type')
```

Comment:
When you documented the API, you specified the field as `auth`, not `auth_type`. Please change the code here to `auth`.